test/test.go

```diff
@@ -505,6 +505,10 @@ func (h *Helper) Path(name string) string {
 		joined = filepath.Join(h.tempdir, name)
 	}
 
+	if _, err := os.Stat(joined); os.IsNotExist(err) {
+		return ""
+	}
+
 	// Ensure it's the absolute, symlink-less path we're returning
 	abs, err := filepath.EvalSymlinks(joined)
```

Comment:
For which test case?  When I remove `final/manifest.json` from a test and run it, I only get a proper "manifest created where none was expected" error from CompareFile.  If this error was ignored, you'd get the effect you're looking for without the need for the above, no?